RR.App/Controllers/HRIS/EmployeeController.cs

```diff
@@ -68,181 +56,61 @@ public async Task<IActionResult> GetEmployeeById([FromQuery] int id)
     [HttpGet("by-email")]
     public async Task<IActionResult> GetEmployeeByEmail([FromQuery] string? email)
     {
-        try
-        {
-            var claimsIdentity = User.Identity as ClaimsIdentity;
-
-            var emailToUse = email ??
-                             claimsIdentity!.FindFirst(ClaimTypes.Email)!.Value;
-
-            var employee = await _employeeService.GetEmployee(emailToUse);
 
+            var employee = await _employeeService.GetEmployee(_identity.Email);
```

Comment:
```suggestion
            var employee = await _employeeService.GetEmployee(email ?? _identity.Email);
```

Missing a crucial part of first using the email that was provided, otherwise use the logged in user's email